README.md

```diff
@@ -42,7 +42,6 @@ To run the app locally, first clone this repository and `cd` into its directory.
 1. Copy the `.env.example` file to `.env`, and edit it including your credentials
```

Comment:
sorry the numbers need to be fixed on this one too :)

Reply:
done! thanks for catching this.